Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt

```diff
@@ -84,6 +85,7 @@ class DartToolingDaemonService private constructor(private val project: Project)
   private fun onServiceStarted(uri: String?, secret: String?) {
     serviceRunning = true
     this.secret = secret
+    this.uri = uri
```

Comment:
Should we set `uri` back to `null` in `processTerminated()`, where `serviceRunning` becomes `false`?

Reply:
Ah yes, that would make sense. Same for in `onClose`?